src/app/pages/datasets/components/dataset-details-card/dataset-details-card.component.html

```diff
@@ -0,0 +1,66 @@
+<div>
+    <mat-card class="container-card">
+      <mat-card-header>
+        <h3 mat-card-title>{{ 'Dataset Details' | translate }}</h3>
+        <button mat-button (click)="editDataset()">{{ 'Edit' | translate }}</button>
+      </mat-card-header>
+      <mat-card-content>
+        <div class="row">
+          <span class="header">Type: </span>
+          <span *ngIf="!loading || !!dataset.type">{{ dataset.type }}</span>
+          <ngx-skeleton-loader
+            *ngIf="loading && !dataset.type"
+            class="loader-skel" 
+          ></ngx-skeleton-loader>
+        </div>
+        <div class="row">
+          <span class="header">Sync: </span>
+          <span *ngIf="!loading || !!dataset.sync?.value">{{ dataset.sync?.value }}</span>
+          <ngx-skeleton-loader
+            *ngIf="loading && !dataset.sync?.value"
+            class="loader-skel" 
+          ></ngx-skeleton-loader>
+        </div>
+        <div class="row">
+          <span class="header">Compression Level: </span>
+          <span *ngIf="!loading || !!datasetCompression">{{ datasetCompression }}</span>
+          <ngx-skeleton-loader
+            *ngIf="loading && !datasetCompression"
+            class="loader-skel" 
+          ></ngx-skeleton-loader>
+        </div>
+        <div class="row">
+          <span class="header">Enable Atime: </span>
+          <span *ngIf="!loading || !!dataset.atime?.value">{{ dataset.atime?.value }}</span>
+          <ngx-skeleton-loader
```

Comment:
It makes sense to wrap `ngx-skeleton-loader` in a component with common styles and etc

Reply:
Let's fix this later. Extracting to `forRoot` should be good enough for now.